### DISCUSSION TITLE
accessing superclasses methods

### DISCUSSION THREAD

**Discussion by SamueleD78** [vfsfitvnm/frida-il2cpp-bridge#165] [Q&A]
so i have a class A which is based on a class B

`class FirstClass : SecondClass`

and SecondClass seems to have a LOT more methods than FirstClass. Those methods are not accessible from a FirstClass instance.
is there is a way to access them or i'm just asking a dumb question? ^^

Thank you in advance, i really appreciate your support

**Comment by vfsfitvnm** [comment]
```ŧs
.method("superclassMethodName"); // it looks for any method in the base class first
.method("superclassMethodName", 2); // specify the parameter count to restrict the query
```
If both base and super classes have a method with the same name and parameter count, then you won't be able to get a reference to the superclass method, see https://github.com/vfsfitvnm/frida-il2cpp-bridge/issues/106

**Reply by SamueleD78** [reply]
i swear i tried and it wasn't working, now it works... it's clear i had made a mistake before.
Thank you for your answer and sorry for wasting your time
